bin/cron_script.sh

```diff
@@ -16,7 +21,7 @@ if [ "$HOST_CLOUD" = 'aws' ]; then
   pip install awscli --upgrade
 
   # Extracting ami ID of latest kubenow stable (usually in the format of "ImageId:" "ami-xxxxxxx",)
-  kubenow_latest_amiId=$(aws ec2 describe-images --filters "Name=name,Values=kubenow-v$SUPPORTED_STABLE_TAG" "Name=owner-id,Values=105135433346" |
```

Comment:
Using the variable SUPPORTED_STABLE_TAG was wrong, it introduced a bug. In fact such variable is mainly related to the execution of the cronjob script. When a build is not triggered by a cronjob, then this `after_success` script still needs to run anyway and has to get the right version of what has been built by packer.